frontity.settings.js

```diff
@@ -16,7 +16,7 @@ const settings = [
         name: "@frontity/wp-source",
         state: {
           source: {
-            api: "https://frontity.org/wp-json",
+            api: "https://dev-frontity-org.pantheonsite.io/wp-json",
```

Comment:
I think this is a mistake, right? :sweat_smile:

Reply:
You're right :smile: I was testing the new version of the Gutenberg plugin.